src/ucs/memory/memtype_cache.h

```diff
@@ -21,6 +21,7 @@ BEGIN_C_DECLS
 
 typedef struct ucs_memtype_cache         ucs_memtype_cache_t;
 typedef struct ucs_memtype_cache_region  ucs_memtype_cache_region_t;
+extern ucs_memtype_cache_t *ucs_memtype_cache_global_instance;
```

Comment:
1. 2 space lines before (it's not a type)
2. add comment to it : `/* The single global instance of the memory type cache */`